### DISCUSSION TITLE
Live Server - Remote HTML Preview

### DISCUSSION THREAD

**Discussion by cleve703** [coder/code-server#2509] [Q&A] [answered]
I am able to use Live Server on my home network to do HTML previews. Is it possible to do this remotely, from the Internet? Note that I'm running code-server on a RaspberryPi.

**Comment by bpmct** [comment, marked as answer]
Yes - this is totally doable. You can use code-server's web proxy to access the ports created by live server: https://github.com/cdr/code-server/blob/v3.8.0/doc/FAQ.md#how-do-i-securely-access-web-services
